.travis.yml

```diff
@@ -19,7 +19,9 @@ cache:
 
 script:
   - ./gradlew build
-  - ./gradlew test jacocoTestReport
+  - ./gradlew test
+  - ./gradlew functional
+  - ./gradlew jacocoTestReport
```

Comment:
move report to `after_success` - to the top of list

Reply:
i've thought about this before. the issue with that is the report may fail to generate but your build stays green. I think we probably want builds to fail if we can't report on the coverage?